src/index.css

```diff
@@ -381,6 +385,7 @@ div#picker {
   padding: 10px 0;
   box-shadow: 0 0 20px rgba(104, 85, 224, 0.2);
   transition: 0.4s;
+  margin-right: 10px;
```

Comment:
that's not the style for logout button, please check and correct.

Reply:
I couldn't find the specific styles for logout button.

Reply:
then you may add a class to that button and update it in the index.css